### PR TITLE
Release  v3.7.0-rc1

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Bump version to v3.7.0-rc1

#### Changes

- Update Changelog and Security
- Bump version.

#### Notes for Reviewers

...

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [ ] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by KrishnaIyer** [comment]
blocked on #2199

File: CHANGELOG.md [hunk 1]
```diff
@@ -666,7 +691,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 NOTE: These links should respect backports. See https://github.com/TheThingsNetwork/lorawan-stack/pull/1444/files#r333379706.
 -->
 
-[unreleased]: https://github.com/TheThingsNetwork/lorawan-stack/compare/v3.6.0...HEAD
+[unreleased]: https://github.com/TheThingsNetwork/lorawan-stack/compare/v3.7.0...HEAD
+[3.7.0]: https://github.com/TheThingsNetwork/lorawan-stack/compare/v3.6.2...v3.7.0
```
Comment (johanstokking) [inline CHANGELOG.md]: Is GitHub smart enough to show a sensible diff here?

Comment (KrishnaIyer) [reply CHANGELOG.md]: No I wasn't sure of this myself. I think I'll just drop the comparison between backports and bumped minor (v3.6.2...v3.7.0).

Comment (KrishnaIyer) [reply CHANGELOG.md]: Would it make sense to compare minor versions here ex: v3.6.0...v3.7.0?

Comment (johanstokking) [reply CHANGELOG.md]: I think in this case it should be `v3.6.0..v3.7.0` yes. I'm afraid there's not much else we can do

**Pull request merged**